.eslintrc.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  parser: '@typescript-eslint/parser',
+  extends: ['plugin:@typescript-eslint/recommended', 'prettier/@typescript-eslint', 'plugin:prettier/recommended'],
+  parserOptions: {
+    ecmaVersion: 2018,
+    sourceType: 'module',
+  },
+  rules: {
+    '@typescript-eslint/no-inferrable-types': 0,
```

Comment:
pls remove

Reply:
this required, otherwise it will change from:
export const SUBMIT_BUTTON_ONE: string = "//*[@id='clickActionSpec']//*[@id='click-button-1']";
to this
export const SUBMIT_BUTTON_ONE = "//*[@id='clickActionSpec']//*[@id='click-button-1']";